.travis.yml

```diff
@@ -14,6 +14,10 @@ matrix:
       env: TOXENV=py36
     - python: 3.6
       env: TOXENV=docs
+    - python: 3.7
+      env: TOXENV=py37
+      dist: xenial
+      sudo: required
```

Comment:
Travis CI is sunsetting their container-based infra so it's going to be VM (sudo: required/true) by default, meaning that this line is probably unnecessary.
```suggestion
```
(And I'd probably recommend switching to xenial by default globally but that's out of scope of this PR)